platform/node/test/js/map.test.js

```diff
@@ -520,17 +513,15 @@ test('Map', function(t) {
             render();
         });
 
-        t.test('throws if called in parallel', function(t) {
+        t.test('error if called in parallel', function(t) {
             var map = new mbgl.Map(options);
             map.load(style);
 
-            t.throws(function() {
-                map.render({}, function() {});
-                map.render({}, function() {});
-            }, /Map is currently rendering an image/);
+            t.plan(1);
```

Comment:
With `t.plan(1)` I believe this will exit after the first test passes. Should probably `t.plan(2)` and assert on the error from the second test instead of a `t.fail()` that won't be reached before the test finishes?